app/views/hyrax/collections/_form.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= simple_form_for @form, html: { class: 'editor' } do |f| %>
+  <div id="descriptions_display">
+    <h2 class="non lower"><%= t('hyrax.collection.form.description') %></h2>
+    <div id="base-terms">
+      <% f.object.primary_terms.each do |term| %>
+        <%= render_edit_field_partial(term, f: f) %>
+      <% end %>
+    </div>
+    <%#= link_to t('hyrax.collection.form.additional_fields'),
+            '#extended-terms',
+            class: 'btn btn-default additional-fields',
```

Comment:
The only reason I can think of keeping it for now is that UI folks would be working on these files to make the UI look better and they can see the option of keeping the 'additional fields' button.